tools/functional-tester/etcd-runner/main.go

```diff
@@ -43,6 +45,7 @@ func main() {
 	endpointStr := flag.String("endpoints", "localhost:2379", "endpoints of etcd cluster")
```

Comment:
@xiang90 One doubt for rounds and eps do you want them to be passed as variables via functions or to directly use them in the functions since they will be global vars?